src/code/audio_heap.c

```diff
@@ -113,350 +113,380 @@ void AudioHeap_DiscardSequence(s32 seqId) {
     }
 }
 
-void AudioHeap_WritebackDCache(void* mem, u32 size) {
-    Audio_WritebackDCache(mem, size);
+/**
+ * Perform a writeback from the L1 data cache to the ram.
+ */
+void AudioHeap_WritebackDCache(void* ramAddr, u32 size) {
+    Audio_WritebackDCache(ramAddr, size);
 }
 
+/**
+ * Attempt to allocated space externally to the audio heap. If no external pool is available,
```

Comment:
```suggestion
 * Attempt to allocate space externally to the audio heap. If no external pool is available,
```
(I'm trying my best to understand what's going on but can't help notice typos)

Reply:
tbh I don't know how these typos keep popping in. I don't think I'm that terrible a writer but yet here they all are :p

Reply:
There's just a lot of new text in here, and it's easy to miss typos when you read your own text you already understand